src/reference/asciidoc/router.adoc

```diff
@@ -959,7 +959,12 @@ As such, please read chapter _<<xml-xpath-routing>>_
 Spring Integration also provides a special type-based router called `ErrorMessageExceptionTypeRouter` for routing Error Messages (Messages whose `payload` is a `Throwable` instance).
 `ErrorMessageExceptionTypeRouter` is very similar to the `PayloadTypeRouter`.
 In fact they are almost identical.
-The only difference is that while `PayloadTypeRouter` navigates the instance hierarchy of a payload instance (e.g., `payload.getClass().getSuperclass()`) to find the most specific type/channel mappings, the `ErrorMessageExceptionTypeRouter` navigates the hierarchy of 'exception causes' (e.g., `payload.getCause()`) to find the most specific `Throwable` type/channel mappings.
+The only difference is that while `PayloadTypeRouter` navigates the instance hierarchy of a payload instance (e.g., `payload.getClass().getSuperclass()`) to find the most specific type/channel mappings,
+the `ErrorMessageExceptionTypeRouter` navigates the hierarchy of 'exception causes' (e.g., `payload.getCause()`)
+to find the most specific `Throwable` type/channel mappings and uses `mappingClass.isInstance(cause)` to match the `cause` to its supper class.
```

Comment:
:smile: I often print this word with two `p` as well as "se**rr**ialization.
Mental block :wink: 

Thank for catching and fix on merge!